### DISCUSSION TITLE
Grid Convergence Angle

### DISCUSSION THREAD

**Discussion by dagheggelund** [DotSpatial/DotSpatial#1529] [Q&A]
I have a question regarding grid convergence angle.   I need to obtain the grid convergence angle for a given projection at a specific location.  Does the DotSpatial project supports obtaining the grid convergence angle for a given projection at a specific location?   
Thanks in advance for any feedback.
Best regards
Dag Heggelund.

**Comment by sindizzy** [comment]
How is that value derived?

**Reply by dagheggelund** [reply]
The following is a reference for Transvers Mercator and Lambert Conformal Conic projections.
https://www.sco.wisc.edu/wp-content/uploads/2020/02/Map-Projection-Functions-WSRS2022.pdf#:~:text=Convergence%2C%20sometimes%20referred%20to%20as%20%E2%80%9Cthe%20mapping%20angle%E2%80%9D%2C,the%20difference%20between%20geodetic%20azimuth%20and%20grid%20azimuth.
Dag

**Reply by dagheggelund** [reply]
One definition of the convergence angle is (from the web)
Convergence, sometimes referred to as “the mapping angle”, also varies from point to point and is the angle between geodetic north and grid north. It is defined as a geodetic azimuth minus the projection of that azimuth on the map projection coordinate grid. Convergence at a point is not the difference between geodetic azimuth and grid azimuth.
